addons/auth_signup/models/res_users.py

```diff
@@ -192,12 +192,20 @@ def _action_reset_password(self, signup_type="reset"):
         self.mapped('partner_id').signup_prepare(signup_type=signup_type)
 
         # send email to users with their signup url
-        account_created_template = None
+        internal_account_created_template = None
+        portal_account_created_template = None
         if create_mode:
-            account_created_template = self.env.ref('auth_signup.set_password_email', raise_if_not_found=False)
-            if account_created_template and account_created_template._name != 'mail.template':
-                _logger.error("Wrong set password template %r", account_created_template)
-                return
+            if any(not user._is_portal() for user in self):
```

Comment:
Would use ``_is_internal`` and ``not is_internal``(that's our main point, internal or not)